### PR TITLE
Additional information for project admins

### PR DESCRIPTION
**Pull request by makeasnek**
- Remove un-needed semicolon
- Fix grammar error in "What's the whitelisting process", turn into ordered list for enhanced readability
- Add section explaining how to calculate the incentive their userbase will receive
- Add section explaining how to increase incentive via rainbymagnitude

### CODE REVIEW

File: wiki/whitelist-process.md [hunk 1]
```diff
@@ -115,10 +115,12 @@ around once every 24 hours. The scrapers and the increase in users can potential
...
 
 ## What's the whitelisting process?
-The first step is making sure you as the admin of the project want to be whitelisted. If you, somewhere public you should say that you were interested and a date when they would want this to happen (anywhere that someone could check such as project forums, Reddit, etc). Following this, find someone to make a poll for you on Gridcoin. Lastly if the community votes votes "yes", then you
-get whitelisted.
 
-## Some things you might want to do beforehand: 
+1. The first step is making sure you as the admin of the project want to be whitelisted. You should state somewhere publicly that you are interested and a date when they would want this to happen (anywhere that someone could check such as project forums, Reddit, etc). 
+2. Find someone to make a poll for you on Gridcoin (or make one yourself)
+3. If the community votes "yes", then you get whitelisted. So far every project put to a whitelist poll has been whitelisted, our users are enthusiastic about contributing to science!
```
Comment (barton2526) [inline wiki/whitelist-process.md:121]: > So far every project put to a whitelist poll has been whitelisted

This is not strictly true. In 2017, Xansons was not voted to be whitelisted (source: https://github.com/Erkan-Yilmaz/GRC/issues/11)

File: wiki/whitelist-process.md [hunk 2]
```diff
@@ -127,6 +129,40 @@ If at any point in the future you want to be removed form the whitelist, you can
...
+
+Each user will receive a portion of the rewards allocated to your project. Their portion is relative to other crunchers on your project. So, for example, if a user completed 10% of your project's total recent average credit, they would receive 10% of the total reward. As a project admin, you can use any system you like for allocating credit so long as it is reasonable. For example, you can allocate extra credit for the quick turnaround of workunits or for certain types of workunits.
+
+Users can then keep these GRC, sell them via an exchange, donate them, or use them to purchase goods or services from vendors which accept GRC.
+
+The formula for figuring out the daily rewards your users will receive in total is:
+
+`(current GRC value) * (28750/ number of whitelisted projects)`
+
+In Oct 2023 when GRC's value is $0.01 USD and there are 16 whitelisted projects, this works out to approximately $530 USD/month. Below are expected monthly rewards in various situations
```
Comment (barton2526) [inline wiki/whitelist-process.md:144]: Do not use fiat for value conversions

File: wiki/whitelist-process.md (hunk 1 above)
Comment (makeasnek) [reply wiki/whitelist-process.md]: fixed

File: wiki/whitelist-process.md [hunk 3]
```diff
@@ -127,6 +129,32 @@ If at any point in the future you want to be removed form the whitelist, you can
...
+
+1. Choose an amount to add to your total incentive. We suggest at least the current total amount of your incentive. So if your project normally receive x GRC per month, make your "rain" x GRC as well.
+2. Let your users know that you will be dispensing additional rewards, the amount of the additional rewards, and when the amount will be "rained". You can do this via an announcement on your site. It's probably a good idea to also post about it on Gridcoin's reddit, discord, and other communities. Giving users a heads up of several weeks will give them time to reconfigure their machines and increase their recent average credit.
```
Comment (barton2526) [inline wiki/whitelist-process.md:151]: nit: Reddit and Discord should be capitalized

File: wiki/whitelist-process.md [hunk 4]
```diff
@@ -127,6 +129,32 @@ If at any point in the future you want to be removed form the whitelist, you can
...
+3. Purchase GRC from an <a href="../guides/acquire-grc.htm">exchange</a> or a community member. If you buy them on an exchange, you'll need to move them to your wallet.
+4. On the day you wish to dispense the rewards, open your Gridcoin wallet and wait until it is fully synced (cloud icon bottom right of wallet).
+5. In the wallet, go to Hamburger menu (top left) -> Help -> Debug Window -> Console
+6. Type in the command `listprojects` to find your project name
+7. Type in the command `rainbymagnitude "your_project_name" amount "message"`. For example: `rainbymagnitude "universe@home" 300 "Thank you for crunching universe!"`
```
Comment (barton2526) [inline wiki/whitelist-process.md:156]: We have not supported messages in rains since 5.3.2

File: wiki/whitelist-process.md (hunk 3 above)
Comment (makeasnek) [reply wiki/whitelist-process.md]: fixed

File: wiki/whitelist-process.md (hunk 4 above)
Comment (makeasnek) [reply wiki/whitelist-process.md]: Fixed

**Review by barton2526 — APPROVED** [review]

**Pull request merged**